packages/flutterfire_cli/lib/src/common/utils.dart

```diff
@@ -234,7 +234,9 @@ end
 
 # Write only if config doesn't exist
 if googleConfigExists == false
-  project.new_file(googleFile)
+  file = project.new_file(googleFile)
+  main_target = project.targets.find { |target| target.name == 'Runner' }
+  main_target.add_file_references([file])
```

Comment:
is it worth adding a check around this line incase main_target could not be found (e.g. they've renamed Runner)

Reply:
Happy to do that check. There is also the option of getting the first file in targets i.e. `project.targets.first`? 

Or prefer the check? 🤔

Reply:
as opposed to `project.targets.find { |target| target.name == 'Runner' }` for clarity